### ISSUE TITLE
Label issues encountered for few text input fields provided on forms page

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4641]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 3.3.2: Labels or Instructions

**Test Environment:**
- Test URL: https://clarity.design/documentation/forms
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Using tab key navigation move focus on to the input fields provided on forms page (horizontal forms layout multi line text input and input fields with helper text).

**Expected result:**
For all the input fields label and it's control type should be announced appropriately along with the value (if any).

**Actual result:**
1. Horizontal  Form layout: Multiline text input field have redundant label announcement as   "Label Label Label Label".
2. Label is not announced for the text input fields which have helper text associated with it.
